health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/Module.js

```
@@ -19,18 +29,104 @@
     language,
     modulePrefix,
   });
+  let user = Digit?.SessionStorage.get("User");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'user' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)